docs/CHANGELOG.rst

```diff
@@ -10,10 +10,20 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 Unreleased
 ==========
 
+Added
+-----
+- Add ``alleyoop-rates`` process
+- Add ``alleyoop-utr-rates`` process
+- Add ``alleyoop-summary`` process
+- Add ``alleyoop-snpeval`` process
+
 Changed
 -------
 - Append sample and genome reference information to the summary output
   file in the ``filtering-chemut`` process
+- **BACKWARD INCOMPATIBLE:** Refactor ``slamdunk-all-paired`` process
```

Comment:
Backward incompatible changes should be placed as a first element in the changelog group (in this case, above "Append sample and genome ..." line